aws/rust-runtime/aws-inlineable/src/presigning.rs

```diff
@@ -186,6 +186,20 @@ pub mod request {
                 .finish()
         }
     }
+
+    impl Into<http::request::Builder> for PresignedRequest {
```

Comment:
let's add an example of using this to the docs—finding this `Into` methods is often challenging